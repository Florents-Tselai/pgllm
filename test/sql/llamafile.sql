select llm_generate('3 neat characteristics of a pelican', 'llamafile');
