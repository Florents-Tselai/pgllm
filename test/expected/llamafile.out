SELECT (jsonb_extract_path_text(llm_generate('3 neat characteristics of a pelican', 'llamafile')::jsonb, 'choices', '0',
                                'message', 'content') LIKE '1. Pelican%');
 ?column? 
----------
 t
(1 row)

